README.md

```diff
@@ -0,0 +1,18 @@
+TaskCluster GitHub Service
+==========================
+
+This service monitors all of the repositories associated with an organization for changes and schedules TaskCluster tasks for any repository which contains a `.taskclusterrc` configuration file. The goal of this project is to provide project owners a method for scheduling jobs in TaskCluster which is quick and straight forward.
+
+##Components
+
+### API Server
+Listens for WebHooks and, if they are valid, forwards them to a pulse exchange.
+
+### Worker
+Listens for WebHook triggered pulse messages and attempts to schedule TaskCluster tasks for any events related to a repository which contains a `.taskclusterrc` file.
+
+##Contributing
+
+###Run Tests
+From the project's base run ``npm test``
+*note:* Initially, this will only run unit tests. To run integration tests add pulse credentals to ``config/test.js``
```

Comment:
Actually, `taskcluster-github.conf.json` will overwrite too... Add it to `.gitignore`.

And yes, we should cleanup config loading I want to use YAML and some smarter stuff of loading from env.